ts/Series/Treegraph/TreegraphSeriesOptions.d.ts

```diff
@@ -25,7 +25,11 @@ import type { PointMarkerOptions } from '../../Core/Series/PointOptions';
 
 
 export type TreegraphLayoutTypes = 'Walker';
-export interface TreegraphSeriesOptions extends OrganizationSeriesOptions {
+export type linkOptions = {
+
+};
+export interface TreegraphSeriesOptions extends TreemapSeriesOptions {
+    link: any;
```

Comment:
instead of `any`, we may create `linkOptionsObject` that will contain all the options for the link. This may also solve the empty object in the `linkOptions` above